shared/versions.yml

```diff
@@ -1,6 +1,133 @@
-# This file is generated by https://github.com/gpii-ops/gpii-version-updater.
+# versions.yml
+#
+# This file is managed by https://github.com/gpii-ops/gpii-version-updater.
+#
+# See the README for details on how to modify which images are used or to add
+# components.
 ---
-
-gpii-dataloader: "gpii/universal@sha256:5101505e0fad784e52442558a794d046f7e1ff321417644e415766deb2b9efb5"
-gpii-flowmanager: "gpii/universal@sha256:5101505e0fad784e52442558a794d046f7e1ff321417644e415766deb2b9efb5"
-gpii-preferences: "gpii/universal@sha256:5101505e0fad784e52442558a794d046f7e1ff321417644e415766deb2b9efb5"
+cert_manager:
+  upstream_image: quay.io/jetstack/cert-manager-controller:v0.6.1
+  image: gcr.io/gpii2test-common-stg/quay.io/jetstack/cert-manager-controller
```

Comment:
I looked at the code and could not find a place where `config[component]["image"]` field is being used. Is it still needed?

Reply:
https://github.com/mrtyler/gpii-infra/blob/gpii-3641-sync-images/shared/rakefiles/vars.rb#L75

Reply:
@mrtyler Referenced code uses `config[component]["generated"]["image"]`, which is populated when [component entry is processed](https://github.com/gpii-ops/gpii-version-updater/pull/13/files#diff-dfd7d6a9d054a8b77b827bb793300c15R32). But my question is about `config[component]["image"]` entries, which I believe added into config manually and not used anywhere.

Reply:
Aha! I thought that diff referred to an earlier version of the file, before i added the `generated` section.

None of those keys (except `upstream_image`) should be there anymore!